src/components/Content/RightPanel/TaskDetails/TaskIcons/TaskIcons.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import './TaskIcons.css';
+import {connect} from 'react-redux';
+import { OverlayTrigger, Tooltip } from 'react-bootstrap';
+import MdArchive from 'react-icons/lib/md/archive';
+import FaCommentsO from 'react-icons/lib/fa/comments-o';
+import FaVideoCamera from 'react-icons/lib/fa/video-camera';
+import { patchTask } from '../../../../../actions';
+import HypothesisLogo from '../../../../../static/hypothesis_logo/HypothesisLogo.png';
+
+const jitsiRoom = 'https://meet.jit.si/PonchoSpiderPageantFoxAlsoLaptopTractorWoundDebrisCaucasianGrapeDishclothFaucetBuddhistRefineryRibbonIridescentWishboneDesktopMugshotLeukemiaOfficeApricotEuthanizeUngloved';
+
+const TaskIcons = ({ gid, patchTask }) => {
+
+  const archiveTask = () => {
+    return patchTask({gid, is_archived: true});
+  }
+
+  const getTooltip = (icon) => {
+    if (icon === 'hypothesis') {
+      return (
+        <Tooltip id="tooltip-hypothesis">
+          <strong>
+            Create Hypothesis Group
+          </strong>
+        </Tooltip>
+      )
+    }
+    if (icon === 'video-chat') {
+      return (
+        <Tooltip id="tooltip-video-chat">
+          <strong>
+            Video Chat
+          </strong>
+        </Tooltip>
+      )
+    }
+    if (icon === 'chat') {
+      return (
+        <Tooltip id="tooltip-text-chat">
+          <strong>
+            Discuss Task
+          </strong>
+        </Tooltip>
+      )
+    }
+    else {
+        return (
+          <Tooltip id="tooltip-archive">
+            <strong>
+              Archive Task
+            </strong>
+          </Tooltip>
+        )
+    }
+  }
+
+  return (
+    <div className="task-icons">
+      <OverlayTrigger placement="bottom" overlay={getTooltip('chat')}>
+        <FaCommentsO size={36} className="icon"/>
+      </OverlayTrigger>
+
+      <OverlayTrigger placement="bottom" overlay={getTooltip('video-chat')}>
+        <a href={jitsiRoom + gid} target="_self" rel="nofollow noreferrer noopener">
```

Comment:
@Moe-Shoman What does `target="_self"` do?

Reply:
Does `tarket="_blank"` make more sense? Hmm...

Reply:
`_self` intends to load the document in the same frame.

Reply:
Okay, can you do `target="_blank"` for now and we'll mess with the popup thing later?

Reply:
Yup, just pushed the change